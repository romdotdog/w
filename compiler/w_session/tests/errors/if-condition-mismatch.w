fn main(a: u64, b: u32, c: u32): u32 {
    if a {
        b
    } else {
        c
    }
}