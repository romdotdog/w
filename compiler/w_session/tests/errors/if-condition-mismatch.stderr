tests/errors/if-condition-mismatch.w:2:8: error: type mismatch
    if a {
       ^
